booster/library/Booster/CLOptions.hs

```diff
@@ -279,6 +283,12 @@ levelToContext =
                 , [ctxt| booster>failure,abort |]
                 ]
             )
+        ,
+            ( "EquationWarnings"
+            ,
+                [ [ctxt| booster>(simplification *|function *)>warning |]
```

Comment:
Is `logWarn` it only for `function`? I thought `simplification` may trigger it, too.